### PR TITLE
Convert cmds `pd` `pdj` `pdJ` to rizin APIs

### PR DESCRIPTION
**Pull request by b1llow**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [ ] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [ ] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

* `pd` `pdj` `pdJ` -> rizin APIs
* update rizin to latest dev

**Test plan (required)**

**Closing issues**

Partially addresses https://github.com/rizinorg/cutter/issues/2666

### CODE REVIEW

**Review by thestr4ng3r — CHANGES_REQUESTED** [review]
The code being disassembled does not seem to change when scrolling through the disasm:

https://user-images.githubusercontent.com/1460997/178110311-2dc30d50-456b-4c4f-b3aa-7eb9c5259bf2.mov

**Comment by b1llow** [comment]
> The code being disassembled does not seem to change when scrolling through the disasm:
> 
> 
> 
> https://user-images.githubusercontent.com/1460997/178110311-2dc30d50-456b-4c4f-b3aa-7eb9c5259bf2.mov
> 
> 
> 
> 
> 
> 
> 
> 

I see it shouldn't have this problem now. @thestr4ng3r

File: src/dialogs/XrefsDialog.cpp [hunk 1]
```diff
@@ -132,7 +132,7 @@ void XrefsDialog::updatePreview(RVA addr)
     tempConfig.set("asm.bytes", false);
 
     // Use cmd because cmRaw cannot handle the output properly. Why?
-    QString disas = Core()->cmd("pd--20 @ " + QString::number(addr));
+    QString disas = Core()->cmd("pd 40 @" + QString::number(addr - 20));
```
Comment (XVilka) [inline src/dialogs/XrefsDialog.cpp]: Is it possible to convert this call as well?

**Review by XVilka — CHANGES_REQUESTED** [review]

File: src/dialogs/XrefsDialog.cpp (hunk 1 above)
Comment (b1llow) [reply src/dialogs/XrefsDialog.cpp]: I updated with a solution.

**Comment by b1llow** [comment]
```
fatal: destination path '/private/var/folders/bc/yvvw2g2s4jlg9r8x7tf61xv800009d/T/woodpecker-local-3165369561/rizinorg/cutter/rizin' already exists and is not an empty directory.
fatal: clone of 'https://github.com/rizinorg/rizin' into submodule path '/private/var/folders/bc/yvvw2g2s4jlg9r8x7tf61xv800009d/T/woodpecker-local-3165369561/rizinorg/cutter/rizin' failed
Failed to clone 'rizin'. Retry scheduled
Cloning into '/private/var/folders/bc/yvvw2g2s4jlg9r8x7tf61xv800009d/T/woodpecker-local-3165369561/rizinorg/cutter/src/translations'...
fatal: destination path '/private/var/folders/bc/yvvw2g2s4jlg9r8x7tf61xv800009d/T/woodpecker-local-3165369561/rizinorg/cutter/rizin' already exists and is not an empty directory.
fatal: clone of 'https://github.com/rizinorg/rizin' into submodule path '/private/var/folders/bc/yvvw2g2s4jlg9r8x7tf61xv800009d/T/woodpecker-local-3165369561/rizinorg/cutter/rizin' failed
Failed to clone 'rizin' a second time, aborting
exit status 1
```

**Comment by XVilka** [comment]
@imbillow please update to the latest Rizin: https://github.com/rizinorg/rizin/commit/b79201b49a0d687c854b6e2f0c88fc8a7f2afa61

**Comment by wargio** [comment]
```
CutterJson.h:73:37: error: no matching function for call to 'rz_json_as_string'
    QString toJson() const { return rz_json_as_string(value); }
```

**Review by wargio — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Review by thestr4ng3r — APPROVED** [review]

**Pull request merged**